codejailservice/app.py

```diff
@@ -56,22 +54,41 @@ def configure_codejail(app):
                 limit_overrides_context=context,
             )
 
+
+# -----------------app configuration-------------------------
+
+app = Flask(__name__)
```

Comment:
I'm not soo familiar with flask, but this general settings seem weird to me in the same app file. Can we move them to a different file that does the config and start?

Reply:
In this commit e4c6724, I propose a change to move routes to their folder. This uses "blueprint" of flask.